.env_sample

```diff
@@ -2,4 +2,8 @@ SLEEP_DATA_PATH=/path/to/sleep/data
 DEBUG=False
 OWL=False
 VIDEO_PATH=/path/to/video
-HATCH_IP=192.168.HATCH.IP
\ No newline at end of file
+HATCH_IP=192.168.HATCH.IP
+AREA_X=
```

Comment:
Nice, when we eventually get around to updating the README I want to mention that users should play around with this. The smaller the area, the less demanding it is on client cpu/gpu. Can be computationally expensive & pointless to pass the entire image through the models

Reply:
As soon as we freeze the modification, I'll comment the code and update all related file (including README) to be aligned with new structure